### PR TITLE
Removed the old phone code, and +46 as a prefix and focus,

### PR DESCRIPTION
**Pull request by Hungryhippoafk**
Reworked/removed code parts
- (+46) in input and the ability to reappear if user removes it and clicks again in input form.
- Removed the old code.

New code.
-Now checks if its either blank  or 10 digits, and checks so there is no space between digits, like 123 344, or letters,symbols.
-Added that the modal3 doesnt open if number is not valid.
-Modal 3 opens only if phone number is blank or 10.

Please review this fix and give feedback if solution is good and works!

### CODE REVIEW

File: javascript/bookroom2.js [hunk 1]
```diff
@@ -123,15 +123,26 @@ document.addEventListener("arrayEvent", (event) => {
 book_form.addEventListener("submit", (event) => {
   event.preventDefault(); // Prevents reloading the page when submit
 
+  const userInputPhone = document.getElementById("phone-number").value.trim();
+
+  // Validate phone number (optional or exactly 10 digits)
+  const isValidPhoneNumber = userInputPhone === "" || /^\d{10}$/.test(userInputPhone);
```
Comment (Nishune) [inline javascript/bookroom2.js:129]: Good solution in my opinion, now its either empty or only numbers =)

File: javascript/bookroom2.js [hunk 2]
```diff
@@ -160,41 +172,25 @@ book_form.addEventListener("submit", (event) => {
...
-    else {
-      document.querySelector("#phone-number").value = "";
-      console.log("Phone number not provided, not an requirement");
-      document.querySelector(".modal3").removeAttribute("id");
-    }
-      
-    });
+
+
+    const userInputPhone = document.getElementById("phone-number").value.trim();
```
Comment (Nishune) [inline javascript/bookroom2.js]: Also good to use trim to remove blankspaces

File: javascript/bookroom2.js [hunk 3]
```diff
@@ -123,15 +123,26 @@ document.addEventListener("arrayEvent", (event) => {
...
+  const userInputPhone = document.getElementById("phone-number").value.trim();
+
+  // Validate phone number (optional or exactly 10 digits)
+  const isValidPhoneNumber = userInputPhone === "" || /^\d{10}$/.test(userInputPhone);
+
+  if (!isValidPhoneNumber) {
+    console.log("Please provide a valid 10-digit phone number or leave it blank.");
+    return; 
+  }
+
```
Comment (Nishune) [inline javascript/bookroom2.js:135]: Maybe remove the console.log before merge here?

**Comment by Nishune** [comment]
I think the code now does what its supposed to do, i also think it was a good solution to remove the (+46), since we dont know if this site is only used in Sweden / by swedes.

I made some comments in the code, the only thing i guess you could remove is the console.log before merge. And maybe we can make a new card in trello to add the console.log as a message to the user in the future =)

Good job Jimmy!

**Comment by Hungryhippoafk** [comment]
@Nishune Excellent! Thanks for the review of code, the comments were helpful! have removed the console.log.

**Review by Nishune — COMMENTED** [review]
Test to send the review

**Pull request merged**